CONTRIBUTING.md

```diff
@@ -57,6 +61,10 @@ The included vscode launch config allows you to launch and attach the debugger i
 file you want to run and start debugging. Note that breakpoint setting for realtime code will be within the
 browser/static directory, not the raw source files, and breakpoints in files under test should work directly.
 
+VSCode allows you to directly launch `package.json` scripts in debug mode.
```

Comment:
This is generic NPM project advice (and specific to one text editor) so I don't think it belongs in the ably-js CONTRIBUTING.md

Reply:
Actually, there is already `VSCode` specific advice right above this. If I had to guess, `vscode` is the main editor used for `js` based development.

Reply:
i don't disagree with anything you said. but still, we don't need to add documentation for how to run npm scripts with vscode to this repo because it's nothing to do with ably-js.

Reply:
Okay, got it. I will remove this. I think it should be pretty common for vscode devs to configure editor properly.

Reply:
done